.Documentation/articles/step-by-step-guides/hello-creator.md

```diff
@@ -0,0 +1,121 @@
+
+## Step 4: Create a Simple Training Application
+
+Let’s take a minute to give an outlook on the training that will be built in this step. Often, we teach trainees to perform a sequence of physical actions in a certain order. Here, we want to teach a Trainee, to:
+
+1. Grab  the sphere (which is visually highlighted)
+2. Place it at a specific position
+
+### Create the Workflow
+
+A right-click with your mouse into an empty area of the ***Workflow Editor*** window displays a contextual menu.
+
+> select `add step`
+
+A `new step` appeared on the Scene. The selected step can be configured in the ***Step inspector*** window. On the very top of the ***Step inspector*** window is the name and description of a step.
+
+> change the name to `Position Sphere`.
+> draw a Connection line between the `initial state` and `Position Sphere`.
+
+### Configuring Steps
+
+A `step` can be configured using `behaviors` and `conditions`. The list of `behaviors` and `conditions` can be extended with little developer effort to meet the needs of your company's training applications. 
+`behaviors` prepare a scene for trainees. `conditions` are actions expected from the trainee to move to the next step. We expect the trainee to grab  the sphere (`condition`), which is visually highlighted (`behavior`), and we expect the trainee to place the sphere (`condition`) at a specific position.
+
+#### Behavior - Highlight the Sphere
+
+> select the ‘Position Sphere’ step in the workflow.
+
+Shift your attention to the opening/already opened ***Step inspector***. You see 3 tabs, ***Behaviors***, ***Transitions***, and ***Unlocked Objects***.
+
+> select the Behavior tab and click the ‘add Behavior’ button.
+> select ‘Highlight Objects’ from the list of Behaviors.
+
+Highlight Object has two properties you can configure, the highlight colour and the Object to highlight.
+
+> drag the sphere object from the hierarchy into the ‘object to highlight’ field.
+
+You get the warning, that ‘Sphere is not configured as IHighlightProperty’ with a ‘Fix-it’ button. The Innoactive Creator simplifies the Training Creation for users not necessarily familiar with unity
+
+> click the fix it button.
+
+Innoactive Creator will take care of configuring the underlying Unity Objects to make highlighting work.
+
+<img src="../images/step-by-step-guides/behavior.jpg" width="400">
+
+ *Figure 3: The configured Highlight object behavior.*
+
+#### Transition – Grab-and-place the sphere
+
+Placing an object implies you 'grabbed' it before. Thus, we integrated both actions into a single condition called ‘snap object’. It’s called ‘snap’ because when trainees approach the target position with the object, they can release the object and it will position and rotate itself at the target position.
+
+> select the 'Transitions' tab.
+
+One `step` can have multiple `transitions` to other `steps`. Since your `step` does not have any, it displays by default ‘Transition to the End of the Chapter’.
```

Comment:
```suggestion
One `step` can have multiple `transitions` to other `steps`. Since your `step` does not have any, it displays by default 'Transition to the End of the Chapter'.
```